### PR TITLE
Add English localization

### PR DESCRIPTION
**Pull request by alorkowski**
Storyboard English localization should automatically be generated when adding a new Storyboard localization. As such, the main changes are the Localizable.strings file and a few "// swiftlint:disable all"  so I could ignore SwiftLint.

### CODE REVIEW

File: BoardBank/MonoBank/AddPlayerViewController.swift [hunk 1]
```diff
@@ -40,7 +41,7 @@ class AddPlayerViewController: UITableViewController, UITextFieldDelegate, UICol
 	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
 		if indexPath.section == 2 && indexPath.row == 0 {
 			let strippedBalance = balanceTextField.text!.components(separatedBy: CharacterSet.decimalDigits.inverted).joined()
-			let name = nameTextField.text!.isEmpty ? "Player" : nameTextField.text!
+			let name = nameTextField.text!.isEmpty ? NSLocalizedString("Player", comment: "") : nameTextField.text!
```
Comment (prrenfrew) [inline BoardBank/MonoBank/AddPlayerViewController.swift]: Put all the localized strings into one central location. A common approach is to use a struct and the localized strings static properties of the struct.

File: BoardBank/MonoBank/AddPlayerViewController.swift [hunk 2]
```diff
@@ -6,6 +6,7 @@
 //  Copyright © 2017 Richard Neitzke. All rights reserved.
 //
 
+// swiftlint:disable all
```
Comment (prrenfrew) [inline BoardBank/MonoBank/AddPlayerViewController.swift]: Defeats the purpose of swiftlint to completely disable it. I don't mind you turning it off for your work until the clean up is done, but don't include it as part of a merge request.

**Comment by prrenfrew** [comment]
@alorkowski left a few comments. Please address these. Closing the PR for now.

**Comment by alorkowski** [comment]
Consolidated string localization and removed SwiftLint disabling comments.

File: BoardBank/MonoBank/Localization/LocalizableStrings.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+//
+//  Created by Alexander Lorkowski on 5/1/19.
+//  Copyright © 2019 Richard Neitzke. All rights reserved.
+//
+
+import Foundation
+
+struct LocalizableStrings {
+  
+  static let transferMoney =
```
Comment (prrenfrew) [inline BoardBank/MonoBank/Localization/LocalizableStrings.swift]: Some of these look like they could be logically combined.

A good way to do that is to have a struct inside this struct defined so you can use it as a namespace.

struct LocalizableStrings {
    struct Generic {
        static let ok = NSLocalizedString("OK", comment: "")
    }
}

File: BoardBank/MonoBank/Localization/en.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+"OK" = "OK";
+"Cancel" = "Cancel";
+"What do you want to do with this player?" = "What do you want to do with this player?";
+"Add" = "Add";
+"Rename" = "Rename";
+"Rename Player" = "Rename Player";
+"Enter a new name for" = "Enter a new name for";
+"Delete" = "Delete";
+"Player" = "Player";
+"Are you sure?" = "Are you sure?";
```
Comment (prrenfrew) [inline BoardBank/MonoBank/Localization/en.lproj/Localizable.strings]: Probably better to give these names similar to how we would give a variable name.

File: BoardBank/MonoBank/Localization/LocalizableStrings.swift (hunk 3 above)
Comment (alorkowski) [reply BoardBank/MonoBank/Localization/LocalizableStrings.swift]: In other words to separate strings of different purpose (i.e informational vs. action)?

**Comment by alorkowski** [comment]
`static let transferMoney =
    NSLocalizedString("", comment: "Ask user how much money to transfer")`
Missing key argument.

File: BoardBank/MonoBank/Localization/LocalizableStrings.swift (hunk 3 above)
Comment (prrenfrew) [reply BoardBank/MonoBank/Localization/LocalizableStrings.swift]: Something like that. Being able to split it up logically so it is easier to figure out why and where they are being used.

**Comment by prrenfrew** [comment]
Looks good to me. @mikemelko @nas539 @stanco2rhyme could one of you guys review this?

**Pull request merged**